mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSearchIndexTest.java

```diff
@@ -227,6 +229,27 @@ void bodySearchShouldNotMatchPhraseOnlyInSubject() throws Exception {
         assertThat(result).isEmpty();
     }
 
+    @Test
+    void dateHeaderParsingShouldNotImpactProcessing() throws Exception {
```

Comment:
You mean, this test is failing?

Reply:
Yes:

```
$ mvn test -Dtest=org.apache.james.mailbox.lucene.search.LuceneMailboxMessageSearchIndexTest#dateHeaderParsingShouldNotImpactProcessing
…
17:35:13,045 [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.477 s <<< FAILURE! - in org.apache.james.mailbox.lucene.search.LuceneMailboxMessageSearchIndexTest
17:35:13,046 [ERROR] dateHeaderParsingShouldNotImpactProcessing  Time elapsed: 0.415 s  <<< ERROR!
org.apache.james.mime4j.field.datetime.parser.TokenMgrError: Lexical error at line 1, column 3.  Encountered: "." (46), after : ""
	at org.apache.james.mailbox.lucene.search.LuceneMailboxMessageSearchIndexTest.dateHeaderParsingShouldNotImpactProcessing(LuceneMailboxMessageSearchIndexTest.java:246)
```

And passes just fine with https://github.com/apache/james-mime4j/pull/88 (though, It's just ignoring the parsing problem).

Btw. what's the general consensus/flow when it comes to fixes in dependent libraries?

Reply:
> Btw. what's the general consensus/flow when it comes to fixes in dependent libraries?

I would say similar to the main project?

Reply:
TokenMgrError shows that lenient parsers are not actually being used here (if I am not wrong).

We need to dig deeper.

Corporate meetings on my side are other, I will have time to take a look today.

Reply:
> I would say similar to the main project?

I meant - would it be correct to bump mime4j dependency version (in james) to `0.8.10-SNAPSHOT` to indicate that during release it should be also included? (though I'm not sure if SNAPSHOTS are published anywhere, so there's that)